CryptoAnalysis/src/main/java/crypto/analysis/CrySLRulesetSelector.java

```diff
@@ -101,4 +117,32 @@ public static CrySLRule makeSingleRule(String rulesBasePath, RuleFormat ruleForm
 		}
 		return rules;
 	}
+	
+	public static List<CrySLRule> makeFromZip(File resource) {
+		List<CrySLRule> rules = Lists.newArrayList();	
+
+		if(resource.exists()) {
+			String resourceFileExtension = Files.getFileExtension(resource.getAbsolutePath());
+			boolean acceptedFormat = false;
+			
+			for(ZipFormat zipFormat : ZipFormat.values()) {
+				if(resourceFileExtension.equals(zipFormat.getFileExtension())) {
+					acceptedFormat = true;
+				}
+			}
+			if(!acceptedFormat) {
+				System.err.println("The format is not supported yet: " + resourceFileExtension);
```

Comment:
I think once Enri's PR is merged we should here throw the new exception and catch that on the consumer to display error messages.

Reply:
I totally agree